src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -23,6 +24,8 @@
     private Label date;
     @FXML
     private CheckBox isDone;
+    @FXML
+    private FlowPane tags;
```

Comment:
1 task will have exactly 1 tag. So it's better to use ```tag``` instead of ```tags``` for the variable name.

Reply:
Okay.

Reply:
I have removed the FlowPane and used a Label instead.